cms/envs/common.py

```diff
@@ -1586,6 +1586,9 @@
     'openedx.core.djangoapps.schedules',
     'rest_framework_jwt',
 
+    # Bulk User Retirement
+    'lms.djangoapps.bulk_user_retirement',
```

Comment:
@pkulkark As this new Django application is not used in the CMS (and properly placed in the `lms/` folder), please remove it as a CMS `INSTALLED_APP`.

Reply:
Done